docs/Clearwater_SNMP_Statistics.md

```diff
@@ -29,6 +29,19 @@ For example, a stat queried at 12:01:33 would display the stats covering:
 
 All latency values are in microseconds.
 
+Many of the statistics listed below are stored in SNMP tables (although the MIB file should be examined to determine exactly which ones). The full table can be retrieved by using the 'snmptable' command. For example, the Initial Registration Stats table for sprout can be retrieved by running:  
```

Comment:
(Super picky) Sprout should be capitalised (`sprout` in lowercase just refers to the binary file), and snmptable should be in \` marks (`snmptable`).